app/services/user.js

```diff
@@ -29,6 +29,26 @@ exports.createUser = async user => {
   }
 };
 
+exports.createOrUpdateUser = async user => {
+  try {
+    logger.info('Finding user to create or update to admin');
+    const userCreatedOrUpdated = await User.findOne({ where: { email: user.email } }).then(obj => {
+      // update
```

Comment:
Remove comments, tThe code should be declarative enough to be self-explanatory

Reply:
comments removed